monitoring/prometheus/alert.rules

```diff
@@ -75,6 +75,14 @@ groups:
           description: Alerts when any of the app instances exceed 70% memory utilisation.
           runbook: https://dfedigital.atlassian.net/wiki/spaces/GGIT/pages/2156036281/App+TTA+Runbook#HighMemory-MEDIUM
           dashboard: https://grafana-prod-get-into-teaching.london.cloudapps.digital/d/0PRnzc2Mk/get-into-teaching-apps?viewPanel=33&orgId=1
+      - alert: GoogleCspViolation
+        expr: 'sum(increase(app_csp_violations_total{blocked_uri=~"google"}[24h])) and hour() == 11 and minute() == 0'
```

Comment:
Just realised this might fire continuously for the minute. Any other way to do this?

Reply:
Yeah thats going to be a problem, also I'm not sure how often alert manager evaluates it may not be every minute so may not fire at all.

I would try something like `... and ON() hour() == 11` in the expression to make it only fire sometime during 11-12 and then look into Alert Manager routes; if you make this alert bubble into a different route (they are all using the default route atm) you should be able to configure it with a `repeat_interval` to be > 1 hour, that way if it fires at some time during 11-12 it can't then fire until the following days 11-12 time period.

https://prometheus.io/docs/alerting/latest/configuration/#route